content/es/.wordlist.txt

```diff
@@ -116,12 +118,14 @@ md
 menu
 Mercurial
 mesh
+methodology
```

Comment:
I think this word is not being used

```suggestion
```

Reply:
NVM, that word appears in another document at:
https://github.com/cncf/glossary/blob/dev-es/content/es/agile-software-development.md?plain=1#L5